client/src/App.js

```diff
@@ -18,6 +19,7 @@ function App () {
         <Route exact path='/' component={NavBar} />
         <Route exact path="/home/:id" component={UserHome} />
         <Route exact path="/rooms" component={Rooms} />
+        <Route exact path="/available/rooms" component={AvailableRooms} />
```

Comment:
the available rooms is the second page after rooms so we can call them like this:
/rooms/available-rooms which '/rooms' refer to previous page

Reply:
Done